### PR TITLE
bc-8 update dependencies

### PR DESCRIPTION
**Pull request by MartinSchuhmacher**
# Description

the dependency update with commit 48d808273d83121c4bc10768dff5eff568a1adaa also included a winston update, that caused the issue.
that is though winston and request-promise share the same transitive dependency: async.
since request-promise is deprecated we should remove it, see bc-93.

## Links to Tickets or other pull requests

https://ticketsystem.hpi-schul-cloud.org/browse/BC-8

## Changes

- dependency updates
- set language for the user while saving rest of the account data

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: .github/workflows/tag.yml [hunk 1]
```diff
@@ -14,23 +14,14 @@ jobs:
         tennens: [default, brb, demo, int, n21, open, thr ]
     steps:
       - uses: actions/checkout@v2
```
Comment (tofixx) [inline .github/workflows/tag.yml:16]: why changing the workflows, how the relate to dependency updates?

File: package.json [hunk 2]
```diff
@@ -61,102 +61,102 @@
...
 		"feedr": "^2.13.5",
 		"font-awesome": "^4.7.0",
 		"form-serializer": "^2.5.0",
-		"generate-password": "^1.5.1",
+		"generate-password": "^1.6.1",
 		"git-repo-info": "^2.1.0",
 		"gulp-change": "^1.0.2",
 		"gulp-cli": "^2.3.0",
 		"gulp-rename": "^2.0.0",
-		"handlebars": "^4.7.6",
```
Comment (tofixx) [inline package.json:91]: what is the dependency which crashed the client? that one should be changed only and it's version should be fixed (to noch change by itself)

**Review by tofixx — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (MartinSchuhmacher) [reply package.json:91]: look into the ticket in the description https://ticketsystem.hpi-schul-cloud.org/browse/BC-8

File: .github/workflows/tag.yml (hunk 1 above)
Comment (Loki-Afro) [reply .github/workflows/tag.yml:16]: ignore it for now, this is not finished yet.

Comment (tofixx) [reply .github/workflows/tag.yml:16]: when this is not finished --> it should not be merged. 
when this is not part of the ticket --> it should be a separate pr.

**Review by Metauriel — APPROVED** [review]

**Pull request closed without merging**